README.md

```diff
@@ -96,13 +93,10 @@ such as pack, place, route, and write output files.)
 
 ### nextpnr-ecp5
 
-For ECP5 support, you must download [Project Trellis](https://github.com/SymbiFlow/prjtrellis),
-then follow its instructions to download the latest database and build _libtrellis_. 
-(for example: `-DTRELLIS_INSTALL_PREFIX=/usr` tells nextpnr to look in `/usr/share/trellis` 
-and `/usr/lib/trellis`) 
+For ECP5 support, install [Project Trellis](https://github.com/SymbiFlow/prjtrellis) to `/usr/local` or another location, which should be passed as `-DTRELLIS_INSTALL_PREFIX=/usr` to CMake. Then build and install `nextpnr-ecp5` using the following commands:
```

Comment:
Mixing `/usr/local` and `/usr` seems a bit confusing here

Reply:
Agree, fixed